time-series-forecast/custom-recipes/time-series-forecast-2-train-evaluate/recipe.json

```diff
@@ -445,16 +445,17 @@
       "label": "Strategy",
       "type": "SELECT",
       "mandatory": true,
-      "description": "Time-based split or cross-validation based on R \"Prophet\" method (slower)",
+      "description": "Time-based split",
```

Comment:
@trivenigandhi can we retain the cross-validation capability of prophet? It would simply be a matter of copying the functions we use from https://github.com/facebook/prophet/blob/master/R/R/diagnostics.R into our evalute.R file? AFAIK it would simply be 4 functions: prophet:::rolling_mean, mae, mape, rmse. This way we can uncomment lines 148-154 of evaluate.R. We will still have to quote prophet, but none of these functions rely on RStan so we are good.

Reply:
Ok, this all makes sense! I will update the R files - but as per your comment further down I think I will directly copy in the functions to avoid having to call prophet at all.